File: CompSci 1/Unit 15, Iteration - do while/GUI10/GUI10.java
import java.awt.*;
import javax.swing.*;

public class GUI10 extends JFrame
{
    // constructor
    public GUI10()
    {
        // set frame attributes
        setLayout(null);
        setSize(500, 500);
        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        getContentPane().setBackground(Color.black);
        setVisible(true);
    }

    public void paint(Graphics page)
    {
        super.paint(page);  // erase background

        int i = 0;
        do {
            if(i % 2 == 0)
            {
                page.setColor(Color.orange);
                page.drawLine(250, 250, (int)(Math.random() * 500), (int)(Math.random() * 500));
            }
            else
            {
                page.setColor(Color.blue);
                page.drawLine(250, 250, (int)(Math.random() * 500), (int)(Math.random() * 500));
            }
            i++;
        }while(i < 300);



    }

    // main method
    public static void main(String[] args)
    {
        GUI10 app = new GUI10();  // run program
    }
}

File: CompSci 1/Unit 15, Iteration - do while/GUI10/GUI10.kt
import java.awt.*
import javax.swing.*

class GUI10 : JFrame() {
    // constructor
    init {
        // set frame attributes
        layout = null
        setSize(500, 500)
        defaultCloseOperation = JFrame.EXIT_ON_CLOSE
        contentPane.background = Color.black
        isVisible = true
    }

    override fun paint(page: Graphics?) {
        super.paint(page)  // erase background

        var i = 0
        do {
            if (i % 2 == 0) {
                page!!.color = Color.orange
                page.drawLine(250, 250, (Math.random() * 500).toInt(), (Math.random() * 500).toInt())
            } else {
                page!!.color = Color.blue
                page.drawLine(250, 250, (Math.random() * 500).toInt(), (Math.random() * 500).toInt())
            }
            i++
        } while (i < 300)


    }
}

fun main(args: Array<String>) {
    GUI10()  // run program
}
